src/components/composer/composer.ts

```diff
@@ -333,6 +336,13 @@ export class Composer extends Component<Props, SpreadsheetEnv> {
     this.dispatch("STOP_EDITION", { cancel: true });
   }
 
+  onCompositionstart() {
+    this.compositionActive = true;
+  }
+  onCompositionend() {
```

Comment:
spell checker is yelling at me: "Compositionstart" isn't a known word
```suggestion
  onCompositionStart() {
    this.compositionActive = true;
  }
  onCompositionEnd() {
```